_includes/tutorials/produce-consume-lang/scala/markup/dev/6_0-consume-narrative.adoc

```diff
@@ -0,0 +1,46 @@
+The following Scala class will be the entry point of the consumer application.
+At `src/main/scala/io/confluent/developer/Consumer.scala` add:
+
++++++
+<pre class="snippet"><code class="java">{%
+    include_raw tutorials/produce-consume-lang/scala/code/src/main/scala/io/confluent/developer/Consumer.scala
+%}</code></pre>
++++++
+
+Lets’s describe the keys pieces of this program.
+
+Reminder: we use an externalised config and this app loads its config from the _consumer_ bloc of the `application.conf`
+
+A `Consumer#conusme` function covers most of the record consumption.
+
+- `Consumer#conusme` takes a
+`https://kafka.apache.org/25/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html[KafkaConsumer]`
+and gives back a collection of `Book`.
+
+- `Consumer#consume` call the `KafkaConsumer#poll` and gets a `ConsumerRecords` filed with as much books as the
+`fetch.max.bytes` Consumer Config allows.
+
+- `Consumer#consume` extracts the value from each records. The consumed books will filled `bookMap`, our book collection
+of type `mutable.Map[String, Book]`.
+
+- We also have two functions `getCount` and `getBooks` just to define two HTTP routes.
```

Comment:
Same thing here put the `Consumer.consume` in a code block with callouts in the narrative.